README.md

```diff
@@ -203,22 +203,21 @@ could affect the position of the inserted elements.
 #### The `remove` hook
 
 Allows you to hook into the removal of an element. The hook is called once a
-vnode is to be removed from the DOM. The handling function recieves both the
+vnode is to be removed from the DOM. The handling function receives both the
 vnode and a callback. You can control and delay the removal with the callback.
-It should be invoked once the hook is done doing its business and the element
+It should be invoked once the hook is done doing its business, and the element
 will only be removed once all `remove` hooks have invoked their callback.
 
-The hook is only triggered when and element is to be removed from its parent –
-not if it is the child of an element that is removed. For that see the destroy
+The hook is only triggered when an element is to be removed from its parent –
+not if it is the child of an element that is removed. For that, see the destroy
 hook.
 
 #### The `destroy` hook
 
-This hook is invoked whenever an element removed from the DOM or the child
+This hook is invoked whenever an element removed from the DOM or from the child
```

Comment:
Could you yourself just put in the words that you think is most correct, or do you prefer that I do the rephrasing according to your suggestion?